### DISCUSSION TITLE
Table of Contents

### DISCUSSION THREAD

**Discussion by mhulse** [jpanther/congo#331] [Ideas]
Two issues:

1. If the ToC is too long, it will overflow off bottom of page without being able to scroll to bottom of the ToC.
2. ToC, when it's at top of page, has border on left and bottom, but not right (seems like it should have border all around)

I'd love to help fix this, but thought I'd kick off discussion first.

**Comment by antomal** [comment]
Yes! Many items do not fit in the browser window and ... disappear at the bottom. There is no way to scroll through the TOC. :-(
